pkg/registry/apis/iam/authorizer.go

```diff
@@ -67,6 +68,26 @@ func newLegacyAuthorizer(ac accesscontrol.AccessControl, store legacy.LegacyIden
 				return []string{fmt.Sprintf("teams:id:%d", res.ID)}, nil
 			}),
 		},
+		accesscontrol.ResourceAuthorizerOptions{
+			Resource: iamv0b.CoreRoleInfo.GetName(),
```

Comment:
This resource authorizer does not work in our case. We use "corerole" instead of "role" here:
https://github.com/grafana/grafana/blob/a37953bb6fcd56d59a076c202eb5c9b1e978c6d0/pkg/services/accesscontrol/authorizer.go#L162